v2/pkg/reporting/trackers/jira/jira.go

```diff
@@ -46,6 +46,8 @@ type Options struct {
 	HttpClient      *retryablehttp.Client
 }
 
+const builderCodeString = "\n{code}\n"
```

Comment:
Same as before. Probably it would be even nicer if we would have a formatter interface and different reporting clients would implement their own methods for code snippets, making text bold or italic etc.

Reply:
Used suggested methods instead of extracting it as a constant. Perhaps we can integrate formatter interface and different reporting clients implementing their own methods for code snippets, making text bold or italic etc. in a separate ticket.